packages/snaps-jest/src/internals/simulation/interface.ts

```diff
@@ -45,33 +46,33 @@ const MAX_FILE_SIZE = 10_000_000; // 10 MB
  */
 export function getInterfaceResponse(
   runSaga: RunSagaFunction,
-  type: DialogType,
+  type: string,
```

Comment:
Maybe create a type for this based on `DIALOG_APPROVAL_TYPES`? Like `keyof DIALOG_APPROVAL_TYPES`.